src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder("Boing");
+        Plane airbus = planeBuilder.setName("airbus")

```

Comment:
seems we are setting the name twice here
I would suggest you to remove the constructor in PlaneBuilder

Reply:
Ok, it was for the test do it replace the name or not, and just check for me how it works. I saw it on the homework video.